### PR TITLE
[readme] Add Python simple example to README

### PR DESCRIPTION
**Pull request by chrisdembia**
This change was suggested by @tkuchida in https://github.com/opensim-org/opensim-core/issues/2216.

Fixes part of #2216

### Brief summary of changes

- This PR adds the Python version of the simple arm example to the README, using the neat collapsible sections that @tkuchida showed in #2216.

### Testing I've completed

- I previewed the README on GitHub.

### Looking for feedback on...

- Should we make the Python and C++ examples more similar (in formatting)?

### CHANGELOG.md (choose one)

- no need to update because...this is a minor change

---
This change is [Reviewable](https://reviewable.io/reviews/opensim-org/opensim-core/2223)

### CODE REVIEW

**Comment by tkuchida** [comment]
> Should we make the Python and C++ examples more similar (in formatting)?

I think it would be good if at least the content were the same (e.g., the C++ code uses `simulate(model, state, 10.0);` whereas the Python code explicitly creates a Manager).

**Comment by chrisdembia** [comment]
> I think it would be good if at least the content were the same (e.g., the C++ code uses simulate(model, state, 10.0); whereas the Python code explicitly creates a Manager).

I don't think `OpenSim::simulate()` is wrapped yet.

File: README.md [hunk 1]
```diff
@@ -28,6 +28,143 @@ Let's simulate a simple arm whose elbow is actuated by a muscle, using
...
 `Bindings/Python/examples/build_simple_arm_model.py`):
 
+<details>
+<summary>Python</summary>
+ 
+```py
+import opensim as osim 
+
+# Define global model where the arm lives.
+arm = osim.Model()
```
Comment (tkuchida) [inline README.md]: To be consistent with the C++ code, this line should be `model = osim.Model("bicep_curl")`.

File: README.md [hunk 2]
```diff
@@ -28,6 +28,143 @@ Let's simulate a simple arm whose elbow is actuated by a muscle, using
...
+arm.setUseVisualizer(True)
+
+# ---------------------------------------------------------------------------
+# Create two links, each with a mass of 1 kg, centre of mass at the body's
+# origin, and moments and products of inertia of zero.
+# ---------------------------------------------------------------------------
+
+humerus = osim.Body("humerus",
+                    1.0,
+                    osim.Vec3(0, 0, 0),
```
Comment (tkuchida) [inline README.md]: Should these be `Vec3(0)` to match the C++ version?

File: README.md [hunk 3]
```diff
@@ -28,6 +28,143 @@ Let's simulate a simple arm whose elbow is actuated by a muscle, using
...
+# Fix the shoulder at its default angle and begin with the elbow flexed.
+shoulder.getCoordinate().setLocked(state, True)
+elbow.getCoordinate().setValue(state, 0.5 * osim.SimTK_PI)
+arm.equilibrateMuscles(state)
+
+# ---------------------------------------------------------------------------
+# Simulate.
+# ---------------------------------------------------------------------------
+
+manager = osim.Manager(arm)
```
Comment (tkuchida) [inline README.md]: There are some additional lines here in the C++ version:
```cpp
// Configure the visualizer.
model.updMatterSubsystem().setShowDefaultGeometry(true);
Visualizer& viz = model.updVisualizer().updSimbodyVisualizer();
viz.setBackgroundType(viz.SolidColor);
viz.setBackgroundColor(White);
```
Should the user assume this configuration is done automatically in Python but not in C++?

**Comment by chrisdembia** [comment]
Thanks @tkuchida. I can make the examples more similar.

**Comment by jimmyDunne** [comment]
@chrisdembia can you check and make any formatting changes to the readme. The Py and Matlab portions run fine.

**Comment by jimmyDunne** [comment]
failure not associated with this PR. Merging.

**Pull request merged**